src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -501,9 +468,10 @@ def parse_unshared_saves(meta, log, fit_methods):
         try:
             meta = parse_s5_saves(meta, log, fit_methods, channel_key)
         except FileNotFoundError:
-            # This channel was skipped or was all masked. Insert NaNs in its place.
+            # This channel was skipped or was all masked.
+            # Insert NaNs in its place.
```

Comment:
Do we want to state in the log file which file(s) was skipped?

Reply:
I'm flexible - this would've already been mentioned in the S5 log, but we can restate that here if desired

Reply:
Ok, I'm fine with not mentioning it here if it's in the S5 log.